module/third_party/proto/tendermint/types/evidence.proto

```
@@ -0,0 +1,68 @@
+syntax = "proto3";
+package tendermint.types;
+
+option go_package = "github.com/tendermint/tendermint/proto/tendermint/types";
+
+import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)